packages/babel-plugin-transform-for-of/src/index.js

```diff
@@ -1,7 +1,7 @@
 import { template, types as t } from "@babel/core";
 
 export default function(api, options) {
-  const { loose } = options;
+  const { loose, forOfAsArray: isForOfAsArray } = options;
```

Comment:
would prefer a little bit more descriptive name - `useForAsArray` or something and that name could be used both as the internal flag and the public plugin option

Reply:
What about just `assmeArray`? After all, `for of` is already specified in the plugin name.

Reply:
@nicolo-ribaudo assumeArray?

Reply:
Yeah, `assumeArray` sounds like a great name for this.

Reply:
yeah, I will update the PR.